zgrab_schema.py

```diff
@@ -743,6 +743,14 @@
     "public_bytes":Binary(),
 })
 
+xssh_signature = SubRecord({
+    "parsed":SubRecord({
+        "algorithm":String(),
+        "value":Binary(),
+    }),
+    "raw":Binary(),
```

Comment:
Yes -- raw is used to store the entire signature structure in certs and for server dh kex messages.

```raw``` is the entire signature structure including the algorithm, value, lengths, and any trailing data.